tekton-chains/overlays/moc/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
```

Comment:
hrmm I see there's a `base`...but it's not being used, is that intentional?

Reply:
oh i missed it, thanks for the catch.